apps/openmw/mwmechanics/aifollow.cpp

```diff
@@ -171,6 +213,16 @@ std::string AiFollow::getFollowedActor()
     return mActorRefId;
 }
 
+void AiFollow::turnActorToFaceTarget(const osg::Vec3f& actorPosition, const osg::Vec3f& targetPosition, AiFollowStorage& storage)
+{
+    osg::Vec3f dir = targetPosition - actorPosition;
+
+    float faceAngleRadians = std::atan2(dir.x(), dir.y());
```

Comment:
Also, IIRC, std::atan2(0, 0) == 0 (atan2 should return NaN only if dir.x() is NaN or dir.y() is NaN).